android/src/main/java/com/visioncameraocr/VisionCameraOcrPackage.kt

```diff
@@ -1,19 +1,25 @@
-package com.visioncameraocr
+ package com.visioncameraocr
 
-import com.facebook.react.ReactPackage
-import com.facebook.react.bridge.NativeModule
-import com.facebook.react.bridge.ReactApplicationContext
-import com.facebook.react.uimanager.ViewManager
-import com.mrousavy.camera.frameprocessor.FrameProcessorPlugin
+ import com.facebook.react.ReactPackage
+ import com.facebook.react.bridge.NativeModule
+ import com.facebook.react.bridge.ReactApplicationContext
+ import com.facebook.react.uimanager.ViewManager
 
+ import com.mrousavy.camera.frameprocessor.FrameProcessorPlugin
+ import com.mrousavy.camera.frameprocessor.FrameProcessorPluginRegistry;
 
-class VisionCameraOcrPackage : ReactPackage {
-    override fun createNativeModules(reactContext: ReactApplicationContext): List<NativeModule> {
-        FrameProcessorPlugin.register(OCRFrameProcessorPlugin())
-        return emptyList()
-    }
+ class VisionCameraOcrPackage : ReactPackage {
+     override fun createNativeModules(reactContext: ReactApplicationContext): List<NativeModule> {
+         if(!OCRFrameProcessorPlugin.isRegistered){
+             OCRFrameProcessorPlugin.isRegistered = true
+             FrameProcessorPluginRegistry.addFrameProcessorPlugin("scanOCR") { _ , options ->
+                 OCRFrameProcessorPlugin(options)
+             }
```

Comment:
I'm facing a compilation error at this line: 
```
e: /home/expo/workingdir/build/node_modules/@ismaelmoreiraa/vision-camera-ocr/android/src/main/java/com/visioncameraocr/OCRFrameProcessorPlugin.kt: (20, 90): No value passed for parameter 'p0'
e: /home/expo/workingdir/build/node_modules/@ismaelmoreiraa/vision-camera-ocr/android/src/main/java/com/visioncameraocr/VisionCameraOcrPackage.kt: (15, 78): Type mismatch: inferred type is ((Mutable)Map<String!, Any!>?, Any?) -> FrameProcessorPlugin! but (((Mutable)Map<String!, Any!>?) -> FrameProcessorPlugin!)! was expected
e: /home/expo/workingdir/build/node_modules/@ismaelmoreiraa/vision-camera-ocr/android/src/main/java/com/visioncameraocr/VisionCameraOcrPackage.kt: (15, 80): Expected one parameter of type (Mutable)Map<String!, Any!>?
e: /home/expo/workingdir/build/node_modules/@ismaelmoreiraa/vision-camera-ocr/android/src/main/java/com/visioncameraocr/VisionCameraOcrPackage.kt: (15, 84): Cannot infer a type for this parameter. Please specify it explicitly.
```

Reply:
I fixed the build error by replacing `_ , options` with `options: MutableMap<String, Any>?`

Reply:
Fixed